a/apicurio-registry/build_info.json

```diff
@@ -0,0 +1,80 @@
+{
+    "package_name" : "apicurio-registry",
+    "github_url":"https://github.com/Apicurio/apicurio-registry",
+    "version": "2.4.1.Final",
+    "package_dir": "a/apicurio-registry",
+    "build_script": "apicurio-registry_ubi_8_5.sh",
+    "validate_build_script": "true",
+
+
+    "2.4.1.Final": {
```

Comment:
@stutiibm  I dont see you are using this "PATCH" variable in build script. Do we really need this. Can we not just use "PACKAGE_VER" variable?

Reply:
Removed PATCH variable.